styleguide/derek/_partials/solutions/header-largeIMAC.ejs

```diff
@@ -1,18 +1,20 @@
-<!--Header Large-->
-<div class="imacBanner-container large">
-  <div class="container">
-    <div class="row">
-      <div class="col-sm-7">
-        <div class="imacBanner-text">
-          <h2 class="imacBanner-headline">3,000 cloud experts working for you 24/7/365?<br/>
-          Sounds like you may need a bigger office.</h2>
-        </div>
+<!--Header Long-->
+<div class="imacBanner large">
+  <div class="imacBanner-container">
+    <div class="imacBanner-textRow">
+      <div class="col-sm-7 imacBanner-textContainer">
```

Comment:
Seems a bit odd to be using the bootstrap class when everything else is maintainable. This may be a situation where `imacBanner-textContainer` should be a unique class for each size banner.